docker/serverless/env-start-esp.py

```diff
@@ -58,6 +67,11 @@ def serve_error_msg(error_msg):
 
 
 def main():
+    # nginx needs these directories to exist, but knative may mount volumes in
+    # /var, hiding the directories created in the Dockerfile.
+    for dirname in ["/var/log/nginx", "/var/cache/nginx"]:
+        ensure_dir(dirname)
```

Comment:
We can do this in start_esp.py so that users can mount /var/log and /var/cache externally for esp image.

Reply:
Ok!